lib/browser/browser.js

```diff
@@ -12,53 +12,33 @@
 /* eslint-disable complexity */
 /* eslint-disable max-statements */
 
-require('./vendor/polyfills');
+require('../vendor/polyfills');
 
 var Q                 = require('q');
-var oauthUtil         = require('./oauthUtil');
-var util              = require('./util');
-var tx                = require('./tx');
-var session           = require('./session');
+var oauthUtil         = require('../oauthUtil');
+var util              = require('../util');
+var builderUtil       = require('../builderUtil');
+var tx                = require('../tx');
+var session           = require('../session');
 var cookies           = require('./cookies');
```

Comment:
is it worth to be consistent to use `storage` which would be override via alias definition.?